src/api/gitHub.js

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env node
 
 import fetch from "node-fetch";
+import https from "https";
 import Ora from "ora";
+const spinner = Ora();
```

Comment:
There seem to be some changes here that have been introduced automatically by your editor, and that makes this pull request quite hard to review. Would you mind reverting them, so this PR is more focused?

Reply:
These changes have been reverted